openpype/hosts/fusion/hooks/pre_fusion_setup.py

```diff
@@ -16,19 +20,80 @@ class FusionPrelaunch(PreLaunchHook):
 
     """
     app_groups = ["fusion"]
+    PROFILE_NUMBER = 16
 
+    def get_fusion_profile_name(self) -> str:
+        """usually set to 'Default', unless FUSION16_PROFILE is set"""
+        return os.getenv(f"FUSION{self.PROFILE_NUMBER}_PROFILE", "Default")
+
+    def get_profile_source(self) -> Path:
+        """Get the Fusion preferences (profile) location.
+        Check https://www.steakunderwater.com/VFXPedia/96.0.243.189/indexad6a.html?title=Per-User_Preferences_and_Paths for reference.
+        """
+        fusion_profile = self.get_fusion_profile_name()
+        fusion_var_prefs_dir = os.getenv(f"FUSION{self.PROFILE_NUMBER}_PROFILE_DIR")
+
+        # if FUSION16_PROFILE_DIR variable exists, return the profile filepath
+        if fusion_var_prefs_dir and Path(fusion_var_prefs_dir).is_dir():
+            fusion_prefs_dir = Path(fusion_var_prefs_dir, fusion_profile)
+            self.log.info(f"Local Fusion prefs environment is set to {fusion_prefs_dir}")
+            fusion_prefs_filepath = fusion_prefs_dir / "Fusion.prefs"
+            return fusion_prefs_filepath
+        
+        # otherwise get the profile from default prefs location 
+        fusion_prefs_path = f"Blackmagic Design/Fusion/Profiles/{fusion_profile}/Fusion.prefs"
+        if platform.system() == "Windows":
+            prefs_source = Path(os.getenv("AppData")) / fusion_prefs_path
+        elif platform.system() == "Darwin":
+            prefs_source = Path("~/Library/Application Support/", fusion_prefs_path).expanduser()
+        elif platform.system() == "Linux":
+            prefs_source = Path("~/.fusion", fusion_prefs_path).expanduser()
+        self.log.info(f"Got Fusion prefs file: {prefs_source}")
+        return prefs_source
+
+    def get_copy_fusion_prefs_settings(self):
+        """Get copy prefserences options from the global application settings"""
+        copy_status = copy_path = None
+        try:
+            copy_status, copy_path, force_sync = get_system_settings()["applications"]["fusion"][
+            "copy_fusion_settings"].values()            
+        except ValueError:
+            self.log.error('Copy prefs settings not found')
+        finally:
+            return copy_status, Path(copy_path).expanduser(), force_sync
```

Comment:
Try won't save you from crashing on `None` value passed to `Path`. Also explicit access to settings keys is preferred.
```suggestion
        copy_fusion_settings = (
            self.data
            ["system_settings"]
            ["applications"]
            ["fusion"]
            .get("copy_fusion_settings", {})
        )
        if not copy_fusion_settings:
            self.log.error("Copy prefs settings not found")
        copy_status = copy_fusion_settings.get("copy_status", False)
        force_sync = copy_fusion_settings.get("force_sync", False)
        copy_path = copy_fusion_settings.get("copy_path") or None
        if copy_path:
            copy_path = Path(copy_path).expanduser()
        return copy_status, copy_path, force_sync
```